wgsl/index.bs

```diff
@@ -14352,6 +14356,13 @@ do not correspond directly to types in WGSL.
 This enables a program to write many densely packed values to memory, which can
 reduce a shader's memory bandwidth demand.
 
+Each builtin applies the *inverse* of a [=channel transfer function=] to several input values, then combines
+their results into a single output value. Teh resul
```

Comment:
Typo
```suggestion
their results into a single output value. The result
```

Reply:
The result seems like a dangling sentence?

Reply:
sigh.  Yep. Removed.